libs/fleet-mcp/tests/contract/test_mcp_tools.py

```diff
@@ -894,3 +894,263 @@ async def test_set_agent_pr_url_validates_agent_exists(agent_server):
                 },
             )
         assert "not found" in str(exc.value).lower()
+
```

Comment:
Remove all backwards compatibility. Only project names like "Setup" and "DataOne" work (not coder-devcontainer)